numbat/src/bytecode_interpreter.rs

```diff
@@ -32,8 +32,7 @@ pub struct LocalMetadata {
 
 #[derive(Debug, Clone)]
 pub struct Local {
-    identifier: CompactString,
-    depth: usize,
+    identifiers: Vec<CompactString>,
```

Comment:
would be nice to deduplicate this with `aliases` inside the `metadata` field, but I'm not sure what's the best way to go about having one field for both